torch/ao/quantization/quantizer/x86_inductor_quantizer.py

```diff
@@ -270,9 +284,32 @@ def set_global(self, quantization_config: QuantizationConfig):
     def set_config_for_operator_type(
         self, operator_type: str, quantization_config: QuantizationConfig
     ):
-        self.operator_type_config[operator_type] = quantization_config
+        warnings.warn(
```

Comment:
you can just remove if you don't have users, the original intention for this is to allow people to set things like: "linear", "linear_relu", ...

Reply:
Thanks and removed.